sdk/eventhubs/azure_messaging_eventhubs/src/in_memory_checkpoint_store.rs

```diff
@@ -35,14 +35,11 @@ impl InMemoryCheckpointStore {
 
     /// Updates the ownership for a specific partition.
     pub fn update_ownership(&self, ownership: &Ownership) -> Result<Ownership> {
-        trace!(
-            "Update ownership for partition {}",
-            ownership.partition_id()
-        );
-        if ownership.partition_id().is_empty()
-            || ownership.event_hub_name().is_empty()
-            || ownership.fully_qualified_namespace().is_empty()
-            || ownership.partition_id().is_empty()
+        trace!("Update ownership for partition {}", ownership.partition_id);
+        if ownership.partition_id.is_empty()
+            || ownership.event_hub_name.is_empty()
+            || ownership.fully_qualified_namespace.is_empty()
+            || ownership.partition_id.is_empty()
         {
             warn!("Ownership is not valid: {:#?}", ownership);
```

Comment:
Nit: Because you're tracing, you might not want to format the struct with "#". It's personal preference, perhaps, but when tracing to the console 1 line per event is easier to read in most cases. Whatever you decide, though. I just noticed this now. I wouldn't hold up the PR for this though even if you decide to change it. It's already big enough. 😉